action.yml

```
@@ -57,7 +57,7 @@ runs:
         echo "🔍 Fetching latest valid Infisical CLI version..."
         
         # Fetch all releases (up to 30 most recent)
-        JSON=$(curl -s "https://api.github.com/repos/Infisical/infisical/releases?per_page=50")
+        JSON=$(curl -s "https://api.github.com/repos/Infisical/infisical/releases?per_page=100")
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)